### PR TITLE
set up files please review them and leave a comment

### PR DESCRIPTION
**Pull request by edgararodriguez**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+<html>
+
+<head>
+<meta charset="utf-8">
+<title>Resume</title>
+<link rel="stylesheet" href="style.css" type="text/css">
+</head>
+<body>
+  <header>
+    <p>Edgar Gasca Rodriguez</p>
```
Comment (calebgmahala) [inline index.html]: this could be a h1 tag

**Review by calebgmahala — APPROVED** [review]
Its good. You are going to have to add some more tags in when doing CSS though just to for warn you

File: index.html (hunk 1 above)
Comment (edgararodriguez) [reply index.html]: k

**Review by daviirose — APPROVED** [review]
Great!!!!!!!!!!!!!

**Pull request merged**